Translates creating-a-sitemap.md (#137)
gatsbyjs/gatsby-pt-BR: docs/docs/creating-a-sitemap.md

---
title: Criando um Sitemap
---

## O que é um sitemap?

> Um [XML sitemap](https://support.google.com/webmasters/answer/156184?hl=en) lista as páginas importantes de um site, assegurando que ferramentas de busca (como o Google) possam encontrar e rastrear todas elas, um sitemap ajuda ferramentas de busca a entender a estrutura do seu site.

Reviewer: ```suggestion
Um [XML sitemap](https://support.google.com/webmasters/answer/156184?hl=en) lista as páginas importantes de um site, assegurando que ferramentas de busca (como o Google) possam encontrar e rastrear todas elas. De maneira geral, um sitemap ajuda ferramentas de busca a entender a estrutura do seu site.
```

}
```

> **Nota:**  A propriedade siteUrl deve ser definida e não deixada em branco.

Reviewer: ```suggestion
**Nota:**  A propriedade `siteUrl` deve ser definida e não deixada em branco.
```

> Em seguida, execute um build (`npm run build`), pois a geração do sitemap ocorrerá apenas para builds de produção. Isso é tudo o que é necessário para obter um sitemap funcional com o Gatsby! Por padrão, o caminho do sitemap gerado é /sitemap.xml e inclui todas as páginas do seu site, mas é claro que o plug-in expõe opções para configurar essa funcionalidade padrão.

Reviewer: ```suggestion
Em seguida, execute um build (`npm run build`), pois a geração do sitemap ocorrerá apenas para builds de produção. Isso é tudo o que é necessário para obter um sitemap funcional com o Gatsby! Por padrão, o caminho do sitemap gerado é `/sitemap.xml` e inclui todas as páginas do seu site, mas é claro que o plugin expõe opções para configurar essa funcionalidade padrão.
```

### Modificações Adicionais

> Passos para modificações adicionais estão disponiveis em [`gatsby-plugin-sitemap` documentation](/packages/gatsby-plugin-sitemap)

Reviewer: ```suggestion
Passos para modificações adicionais estão disponiveis na [documentação do `gatsby-plugin-sitemap`](/packages/gatsby-plugin-sitemap)
```

## Mais informações

> - Confira também esta postagem em [gatsby-plugin-advanced-sitemap](/blog/2019-05-07-advanced-sitemap-plugin-for-seo/) do Blog do Gatsby

Reviewer: ```suggestion
- Confira também esta postagem sobre [gatsby-plugin-advanced-sitemap](/blog/2019-05-07-advanced-sitemap-plugin-for-seo/) no blog do Gatsby
```
